db-template/python/geospatial.py

```diff
@@ -71,7 +73,7 @@ def pad_map_bounds(
 def map_simple(
     data: gpd.GeoDataFrame,
     projection: str = NYC_PROJECTION,
-    basemap: str | TileProvider = DEFAULT_BASEMAP,
+    basemap: Union[str, TileProvider] = DEFAULT_BASEMAP,
```

Comment:
`|` to indicate multiple types in a typehint is only support in python versions >=3.10